app/views/unattended/provisioning_templates/PXEGrub2/autoyast_default_pxegrub2.erb

```diff
@@ -0,0 +1,62 @@
+<%#
+kind: PXEGrub2
+name: AutoYaST default PXEGrub2
+model: ProvisioningTemplate
+oses:
+- SLES
+- OpenSUSE
+description: |
+  The template to render PXEGrub2 bootloader configuration for AutoYaST based distributions.
+  The output is deployed on the host's subnet TFTP proxy.
+-%>
+# This file was deployed via '<%= template_name %>' template
+<%
+  if !@host.pxe_loader_efi?
+    linuxcmd = "linux"
+    initrdcmd = "initrd"
+  else
+    linuxcmd = "linuxefi"
+    initrdcmd = "initrdefi"
+  end
+  http_proxy_string = host_param('http-proxy') ? "proxy=http://" + host_param('http-proxy') + ":" + host_param('http-proxy-port') : ''
+  kernel_options = "ramdisk_size=65536 install=#{@mediapath} autoyast=#{foreman_url('provision')} textmode=1 #{http_proxy_string}"
+-%>
+
+set default=<%= host_param('default_grub_install_entry') || 0 %>
+set timeout=<%= host_param('loader_timeout') || 10 %>
+
+menuentry '<%= template_name %>' {
+  <%= linuxcmd %> <%= @kernel %> <%= kernel_options %>
+  <%= initrdcmd %> <%= @initrd %>
+}
```

Comment:
This is good, is this tested?

The rest is a bit crazy stuff for UEFI HTTP Boot, have you tried this? If you just copy pasted it here, I would rather not add it for now, until someone needs it.

Reply:
We did test this before opening the MR, it is what we currently use for UEFI environments